src/components/Table/helpers/renderDetailsLine.js

```diff
@@ -0,0 +1,39 @@
+import React, { Fragment } from 'react'
+import { isNull, isUndefined, isFunction, size } from 'lodash'
+
+import chain from 'helpers/generators/chain'
+import { StyledTableLine, StyledTableCell } from '../styles'
+
+const renderDetailsLine = (
+  cols,
+  element,
+  id,
+  odd,
+  { isDetailsLineHoverable, renderDetailsLine },
```

Comment:
I reckon you can add a `{ isDetailsLineHoverable, renderDetailsLine }= {}` default value, or the object parameter will be considered as required.